public/js/lib/spell-checker/tutorial.js

```diff
@@ -0,0 +1,39 @@
+import introJs from 'intro.js'
+import 'intro.js/introjs.css'
+
+export function initTutorial () {
+  const spellCheckToggle = document.querySelector('.status-spellcheck')
+  const spellCheckStatus = document.querySelector('.spell-check-status')
+
+  if (!spellCheckToggle || !spellCheckStatus) {
+    return
+  }
+
+  introJs().setOptions({
+    dontShowAgain: true, // Override styles to hide the checkbox
+    nextLabel: 'Suivant',
+    prevLabel: 'Précédent',
+    doneLabel: 'Terminé',
+    highlightClass: 'custom-introjs-helperLayer',
+    showBullets: false,
+    steps: [
+      {
+        title: 'Nouveauté',
+        intro: "La correction orthographique est disponible en version bêta. Vous pouvez l'activer ou la désactiver en cochant cette case.",
+        element: spellCheckToggle
+      },
+      {
+        title: 'Feedbacks',
+        intro: "Ce statut vous informera de l'état de votre document. Une fois le tutoriel terminé, cliquez dessus à tout moment pour partager vos feedbacks.",
+        element: spellCheckStatus
+      }
+    ]
+  }).start()
+  // Set a cookie to prevent showing the tutorial again
+  // This ensures the tutorial is shown only once
+  introJs().setDontShowAgain(true)
```

Comment:
How about:

- setDontShowAgain only [oncomplete](https://introjs.com/docs/tour/api#introjsoncompleteprovidedcallback) and onexit?
- set [exitOnOverlayClick](https://introjs.com/docs/tour/options#exitonoverlayclick) to `false` so that a misclick doesn't make us miss the tutorial? It has a close button, only two steps, and we can press esc. I feel the overlay click is more misleading than helpful.